src/main/java/trackie/tasks/Task.java

```diff
@@ -0,0 +1,42 @@
+package trackie.tasks;
+
+public abstract class Task {
+    protected String description;
+    protected boolean completed;
```

Comment:
You could consider naming this boolean something like `isCompleted` so that it sounds like a boolean.